server/routes.js

```diff
@@ -1,4 +1,5 @@
 const server = require("./graderoom.js");
+const fs = require("fs")
```

Comment:
Is this necessary?

Reply:
My understanding is that the way IOS works is that it looks for the icon on that specific route that I added. I had to import fs (file system) to serve the image over that route. There might be a better way to do it but that made the more sense at the time

Reply:
You don't use this though. You use `res.sendFile()`

Reply:
Oh you're right, I didn't end up using it. I'll go ahead and remove it.